### ISSUE TITLE
Uncaught TypeError: cleanedIds is not iterable

### ISSUE THREAD

**Issue by pontusn** [MicrosoftEdge/WebViewFeedback#157]
After upgrading to 84.0.484.0 we started seeing a new error in the console:

```
Uncaught TypeError: cleanedIds is not iterable
    at LocalRefTracker._cleanup (<anonymous>:1:1533)
```

Relatede code:
```
                    _cleanup(cleanedIds) {
                        for (const cleanedId of cleanedIds) {
                            this._log("cleanup for " + cleanedId);
                            this._idToCallback[cleanedId]();
                            delete this._idToCallback[cleanedId];
                        }
                    }
```

I even managed to set a breakpoint inside. The argument `cleanedIds` is nothing but an integer. The function is at the top of the callstack.

**Comment by david-risney** [comment]
Thanks! We'll look into this.

**Comment by cglacken** [comment]
I am also having this same issue

**Comment by champnic** [comment]
This should be fixed in Edge 84.0.495.0+. Thanks!
